homeassistant/components/fritzbox_netmonitor/sensor.py

```diff
@@ -96,20 +96,20 @@ def state(self):
     def state_attributes(self):
         """Return the device state attributes."""
         # Don't return attributes if FritzBox is unreachable
-        if self._state == STATE_UNAVAILABLE:
-            return {}
-        attr = {
-            ATTR_IS_LINKED: self._is_linked,
-            ATTR_IS_CONNECTED: self._is_connected,
-            ATTR_EXTERNAL_IP: self._external_ip,
-            ATTR_UPTIME: self._uptime,
-            ATTR_BYTES_SENT: self._bytes_sent,
-            ATTR_BYTES_RECEIVED: self._bytes_received,
-            ATTR_TRANSMISSION_RATE_UP: self._transmission_rate_up,
-            ATTR_TRANSMISSION_RATE_DOWN: self._transmission_rate_down,
-            ATTR_MAX_BYTE_RATE_UP: self._max_byte_rate_up,
-            ATTR_MAX_BYTE_RATE_DOWN: self._max_byte_rate_down,
-        }
+        attr = {}
+        if self._state != STATE_UNAVAILABLE:
```

Comment:
We add indentation like this. It's better to keep it a guard clause. We can return twice if we want to remove a variable.

Reply:
Sure thing! Adjusted the code accordingly.